Image_process/processImage.py

```diff
@@ -0,0 +1,42 @@
+from PIL import Image
+
+
+class ImageProcess:
+    def __init__(self, image_path):
+        self.image_path = image_path
+
+    def resize_image(self):
+        """
+        Resize the image to the specified width and height.
+        :param width: Target width of the image.
+        :param height: Target height of the image.
+        :return: Resized Image object.
+        """
+        width = 128
```

Comment:
width and height should be parameters of the method, setting the default values to 128x128 makes sense, so something like

```python
def resize_image(self, height: int = 128, width: int = 128):
    if height <= 0 or width <= 0:
        raise ValueError("height or width less than zero")
```